apps/meteor/ee/app/livechat-enterprise/server/lib/requestPdfTranscript.ts

```diff
@@ -1,12 +1,15 @@
 import { OmnichannelTranscript, QueueWorker } from '@rocket.chat/core-services';
 import type { AtLeast, IOmnichannelRoom } from '@rocket.chat/core-typings';
-import { LivechatRooms } from '@rocket.chat/models';
+import ExpiryMap from 'expiry-map';
 
 import { logger } from './logger';
 
+// Allow to request a transcript again after 15 seconds, assuming the first one didn't complete
```

Comment:
```suggestion
// Allow to request a transcript again after 15 seconds, assuming the first one didn't complete
// This won't prevent multiple transcript generated for the same room in a multi-instance deployment since state is not shared, but we're ok with the drawbacks
```